spec/rubocop/cop/performance/use_zip_to_wrap_array_contents_spec.rb

```diff
@@ -0,0 +1,153 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Performance::UseZipToWrapArrayContents, :config do
+  context 'when using map with array literal' do
+    it 'registers an offense and corrects to use zip with no arguments' do
+      expect_offense(<<~RUBY)
+        [1, 2, 3].map { |id| [id] }
+                  ^^^ Use `.zip` instead of `.map { |id| [id] }`.
```

Comment:
```suggestion
                  ^^^^^^^^^^^^^^^^^ Use `.zip` instead of `.map { |id| [id] }`.
```

I think this should include the block as well

Reply:
Okay, I got something together that covers both the method call and its block